src/Cbor.h

```diff
@@ -38,7 +40,8 @@ class Encode {
     /// encode an array of elements (of different types)
     static Encode array(const std::vector<Encode>& elems);
     /// encode a map
-    static Encode map(const std::vector<std::pair<Encode, Encode>>& elems);
+    //static Encode map(const std::vector<std::pair<Encode, Encode>>& elems);
+    static Encode map(const std::map<Encode, Encode>& elems);
```

Comment:
Does the map need to be ordered? Can we use `std::unordered_map` instead?

Reply:
For the map it's better if order is deterministic, better for testing.